freanalysis_land/freanalysis_land/land.py

```diff
@@ -0,0 +1,243 @@
+import json
+from analysis_scripts import AnalysisScript
+import intake
+import matplotlib.pyplot as plt
+import cartopy
+import cartopy.crs as ccrs
+import datetime
+import pandas as pd
+import re
+import xarray as xr
+
+
+class LandAnalysisScript(AnalysisScript):
+    """Abstract base class for analysis scripts.  User-defined analysis scripts
+       should inhert from this class and override the requires and run_analysis methods.
+
+    Attributes:
+       description: Longer form description for the analysis.
+       title: Title that describes the analysis.
+    """
+    def __init__(self):
+        """Instantiates an object.  The user should provide a description and title."""
+        self.description = "This is for analysis of land model (stand-alone)"
+        self.title = "Soil Carbon"
+
+    def requires(self):
+        """Provides metadata describing what is needed for this analysis to run.
+
+        Returns:
+            A json string describing the metadata.
+        """
+        raise NotImplementedError("you must override this function.")
```

Comment:
if the `return` on L33 is what you want, remove this `NotImplementedError`

Reply:
I was not taking advantage of this function for this version of the analysis, so this function is never called. I could comment it out for this implemenatation, perhaps?